CONTRIBUTING.md

```diff
@@ -327,9 +327,10 @@ checker, and leave out unnecessary detail:
 
 Some further tips for good type hints:
 * use built-in generics (`list`, `dict`, `tuple`, `set`), instead
-  of importing them from `typing`, **except** for arbitrary length tuples
-  (`Tuple[int, ...]`) (see
-  [python/mypy#9980](https://github.com/python/mypy/issues/9980));
+  of importing them from `typing`, **except** in type aliases and for
+  arbitrary length tuples (`Tuple[int, ...]`);
```

Comment:
Apparently base classes didn't make it to the previous line. I don't think inheriting from `tuple[int, ...]` works.

```suggestion
  of importing them from `typing`, **except** in type aliases, in base classes, and for
  arbitrary length tuples (`Tuple[int, ...]`);
```